tests/functional/settings_change_email.js

```diff
@@ -0,0 +1,144 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+define([
+  'intern',
+  'intern!object',
+  'tests/lib/helpers',
+  'tests/functional/lib/helpers',
+  'tests/functional/lib/selectors',
+], function (intern, registerSuite, TestHelpers, FunctionalHelpers, selectors) {
+
+  const config = intern.config;
+  const SIGNUP_URL = config.fxaContentRoot + 'signup';
+  const PASSWORD = 'password';
+  const NEW_PASSWORD = 'password1';
+
+  let email;
+  let secondaryEmail;
+
+  const clearBrowserState = FunctionalHelpers.clearBrowserState;
+  const click = FunctionalHelpers.click;
+  const fillOutChangePassword = FunctionalHelpers.fillOutChangePassword;
+  const fillOutResetPassword = FunctionalHelpers.fillOutResetPassword;
+  const fillOutCompleteResetPassword = FunctionalHelpers.fillOutCompleteResetPassword;
+  const fillOutSignIn = FunctionalHelpers.fillOutSignIn;
+  const fillOutSignUp = FunctionalHelpers.fillOutSignUp;
+  const openPage = FunctionalHelpers.openPage;
+  const openVerificationLinkInNewTab = FunctionalHelpers.openVerificationLinkInNewTab;
+  const openVerificationLinkInSameTab = FunctionalHelpers.openVerificationLinkInSameTab;
+  const testIsBrowserNotified = FunctionalHelpers.testIsBrowserNotified;
+  const testElementExists = FunctionalHelpers.testElementExists;
+  const testElementTextEquals = FunctionalHelpers.testElementTextEquals;
+  const testErrorTextInclude = FunctionalHelpers.testErrorTextInclude;
+  const testSuccessWasShown = FunctionalHelpers.testSuccessWasShown;
+  const type = FunctionalHelpers.type;
+  const visibleByQSA = FunctionalHelpers.visibleByQSA;
+
+  registerSuite({
+    name: 'settings change email',
+
+    beforeEach: function () {
+      email = TestHelpers.createEmail();
+      secondaryEmail = TestHelpers.createEmail();
+      return this.remote.then(clearBrowserState())
+        .then(openPage(SIGNUP_URL, selectors.SIGNUP.HEADER))
+        .then(fillOutSignUp(email, PASSWORD))
+        .then(testElementExists('#fxa-confirm-header'))
```

Comment:
Since this is a new suite, can you replace all the inline selectors w/ selectors from [selectors.js](https://github.com/mozilla/fxa-content-server/blob/8fafad7f418419c3eea679e345301e03a43c790c/tests/functional/lib/selectors.js), and if the appropriate selector isn't available, add it?